extensions/ql-vscode/src/helpers.ts

```diff
@@ -266,6 +269,36 @@ export function isFolderAlreadyInWorkspace(folderName: string) {
   );
 }
 
+/** Check if the current workspace is the CodeTour and open the workspace folder.
+ * Without this, we can't run the code tour correctly.
+ **/
+export async function prepareCodeTour(): Promise<void> {
+  if (workspace.workspaceFolders?.length) {
+    const currentFolder = workspace.workspaceFolders[0].uri.fsPath;
+
+    // We need this path to check that the file exists on windows
+    const tutorialWorkspacePath = join(
+      currentFolder,
+      "tutorial.code-workspace",
+    );
+    const toursFolderPath = join(currentFolder, ".tours");
+
+    if (
+      existsSync(tutorialWorkspacePath) &&
+      existsSync(toursFolderPath) &&
+      !isCodespacesTemplate()
+    ) {
+      const tutorialWorkspaceUri = Uri.parse(
+        join(
+          workspace.workspaceFolders[0].uri.fsPath,
```

Comment:
It should be, but unfortunately I've found the two behave differently for windows tests. It might be related to the error you found above. I'll look into it.

Reply:
Ended up using the whole path, which we already store: https://github.com/github/vscode-codeql/pull/2188/commits/737a1f5c370940876357548fdcfd59f966874ce6